data/Madhavi210.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Madhavi Joshi",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Learning Full Stack Web Development",
+  "avatar": "https://github.com/Madhavi210.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Madhavi_Joshi__",
+      "icon": "FaTwitter"
+    }
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/madhavi-joshi-0b3228239/",
+      "icon": "FaLinkedIn"
```

Linkedin icon name should be `FaLinkedin`

```suggestion
      "icon": "FaLinkedin"
```